packages/core/integration-tests/test/html.js

```diff
@@ -1058,7 +1058,7 @@ describe('html', function() {
         __dirname,
         '/integration/html-js-shared-dynamic-nested/index.html',
       ),
-      {production: true, scopeHoist: true},
+      {mode: 'production', production: true, scopeHoist: true},
```

Comment:
Can you remove these `production: true` properties? These are leftovers from Parcel 1. And looks like these tests actually once ran in production mode 😄

Reply:
Yes! I wasn't sure if that should be a separate task, but I'm on it 🙇‍♀️  @mischnic

Reply:
I resolved this and updated the branch but I'm not sure why the checks aren't passing 🤧  @mischnic